README.md

```diff
@@ -113,3 +113,30 @@ module.exports.GLYPHS = GLYPHS;
 var Icon = require('components/icon');
 <Icon glyph={Icon.GLYPHS.UNICORN}>
 ```
+
+Usage with React 0.14
+
+```js
+// icon.jsx
+const Icon = ({glyph, width, height, className}) => (
+  <svg className={className} width={width} height={height}>
+    <use xlinkHref={`${glyph}`} />
+  </svg>
+);
+
+Icon.defaultProps = {
+  width: 16,
+  height: 16,
+  className: 'icon'
+}
+
+export default Icon;
```

Comment:
Hmm. The whole thing can be replaced with:

```
export default ({glyph, width = 16 , height = 16, className = 'icon'}) => (
  <svg className={className} width={width} height={height}>
    <use xlinkHref={glyph} />
  </svg>
);
```
Doesn't show up as "Icon" in the devtools, though.